src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1010,24 +1010,23 @@ private CmdletInfo GetNextCmdlet()
                         return null;
```

Comment:
@powercode, your last commit had 22 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Validate that get-help works for AllUsers Scope.for module : Microsoft.PowerShell.Core.Validate -Description and -Examples sections in help content. Run 'Get-help -name <cmdletName>
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Help/HelpSystem.Tests.ps1: line 138
138:             $help = Get-Help -Name $cmdletName

```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @('Add-History', 'Clear-History', 'Debug-Job', 'Disable-ExperimentalFeature', 'Enable-ExperimentalFeature', 'Enter-PSHostProcess', 'Enter-PSSession', 'Exit-PSHostProcess', 'Exit-PSSession', 'Export-ModuleMember').
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Remove-Alias; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-Location; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-MarkdownOption; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-PSBreakpoint; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-PSDebug; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-StrictMode; ConfirmImpact=None; SideIndicator==>}, @{Name=Set-TraceSource; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Set-Variable; ConfirmImpact=Medium; SideIndicator==>}, @{Name=Show-Markdown; ConfirmImpact=None; SideIndicator==>}, @{Name=Sort-Object; ConfirmImpact=None; SideIndicator==>}).
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```
File encoding tests.ParameterType for parameter 'Encoding' should be 'Encoding'.Encoding parameter of command '<Command>' is type 'Encoding'
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Basic/Encoding.Tests.ps1: line 14
14:             $command.Parameters['Encoding'].ParameterType.FullName | Should -BeExactly "System.Text.Encoding"

```
Validate that get-help works for CurrentUserScope.for module : Microsoft.PowerShell.Core.Validate -Description and -Examples sections in help content. Run 'Get-help -name <cmdletName>
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Help/HelpSystem.Tests.ps1: line 94
94:             $help = Get-Help -Name $cmdletName

```